src/EfeCapCond.m

```diff
@@ -105,3 +96,17 @@
             end
         end
     end
+
+function kerstenCoef = calcKerstenCoef(theta, porosity)
+%{
+    Calculate the kersten coefficient, weighting dry and wet thermal conductivity
+    See Xyz (20xx), page 12.

```

Comment:
You still need to add a proper reference here!

Reply:
@yijianzeng @Yunfei-Wang1993 can you address this comment? so the pull request can be finalized.

Reply:
Thanks a lot, Bart, Sarah, the reference is as below:
Jansson, P. E. (2012). CoupModel: Model use, calibration, and validation. Transactions of the ASABE, 55(4), 1337. https://doi.org/10.13031/2013.42245

Apologize for the late though.